backend/canisters/community/impl/src/queries/selected_updates.rs

```diff
@@ -27,11 +27,17 @@ fn selected_updates_impl(args: Args, state: &RuntimeState) -> Response {
     let invited_users_last_updated = data.invited_users.last_updated();
     let events_last_updated = data.events.latest_event_timestamp();
     let user_groups_last_updated = data.members.user_groups_last_updated();
-
-    let last_updated = [invited_users_last_updated, events_last_updated, user_groups_last_updated]
-        .into_iter()
-        .max()
-        .unwrap();
+    let display_names_last_updated = data.members.display_names_last_updated();
```

Comment:
Iterating over every member doesn't seem right. We should maintain a `last_updated` field instead.